tunnel/config.go

```diff
@@ -18,7 +18,8 @@ type SSHConfigFile struct {
 
 // NewSSHConfigFile creates a new instance of SSHConfigFile based on the given
```

Comment:
Please update the documentation to describe the function is looking for the ssh configuration file on `$HOME/.ssh/config` instead of receiving the path as an argument.